.gitignore

```diff
@@ -3,3 +3,4 @@ npm-debug.log
 .DS_Store
 .idea
 microjs.com.iml
+package-lock.json
```

Comment:
Please only change `data.js` when adding a library. Please do not change `.gitignore`.